### ISSUE TITLE
snaps cannot be used

### ISSUE THREAD

**Issue by eraclitux** [scaleway/image-ubuntu#72]
Hi,
your kernels lack support for some options to make Ubuntu snaps works.
es:
`# CONFIG_SQUASHFS_XZ is not set`
and maybe other parameters.

Complete error on **Xenial** running `snap install  <snap>` on a VC1 instance:
```
error: cannot perform the following tasks:
- Mount snap "ubuntu-core" ([start snap-ubuntu\x2dcore-122.mount] failed with exit status 1: Job for snap-ubuntu\x2dcore-122.mount failed. See "systemctl status "snap-ubuntu\\x2dcore-122.mount"" and "journalctl -xe" for details.
)
```
`journalctl -xe` output:
```
-- Unit snap-ubuntu\x2dcore-122.mount has begun starting up.
Jul 03 11:40:26 poseidon.safehouse.link mount[26997]: mount: wrong fs type, bad option, bad superblock on /dev/loop0,
Jul 03 11:40:26 poseidon.safehouse.link mount[26997]:        missing codepage or helper program, or other error
Jul 03 11:40:26 poseidon.safehouse.link mount[26997]:        In some cases useful info is found in syslog - try
Jul 03 11:40:26 poseidon.safehouse.link mount[26997]:        dmesg | tail or so.
Jul 03 11:40:26 poseidon.safehouse.link kernel: squashfs: SQUASHFS error: Filesystem uses "xz" compression. This is not supported
Jul 03 11:40:26 poseidon.safehouse.link systemd[1]: snap-ubuntu\x2dcore-122.mount: Mount process exited, code=exited status=32
Jul 03 11:40:26 poseidon.safehouse.link systemd[1]: Failed to mount Mount unit for ubuntu-core.
```
It would be great if you could enable the ability to run snaps.
Have a nice day.

**Comment by tscolari** [comment]
Is this fix also in place for the VC servers? I'm having the exactly same error when the snap tries to mount something.

**Comment by nikos9742** [comment]
Same error still here ... @Scaleways ?

**Comment by m-vellinga** [comment]
Having the same issue. This is not working on VC1S:

```
Apr 19 20:21:17 mount[11619]: mount: wrong fs type, bad option, bad superblock on /dev/loop0,
Apr 19 20:21:17 mount[11619]:        missing codepage or helper program, or other error
Apr 19 20:21:17 mount[11619]:        In some cases useful info is found in syslog - try
Apr 19 20:21:17 mount[11619]:        dmesg | tail or so.
Apr 19 20:21:17 kernel: squashfs: version 4.0 (2009/01/31) Phillip Lougher
Apr 19 20:21:17 kernel: squashfs: SQUASHFS error: Filesystem uses "xz" compression. This is not supported
```
Referenced issue does not show any changes in the kernel that this image usage (4.4.38-std-1)

**Comment by wizzard0** [comment]
Yep, also does not work on C1 (Xenial)

**Comment by guillaumedsde** [comment]
Does not work for me either, same problem (no XZ compression, perhaps issue should be reopened?

**Comment by tscolari** [comment]
I believe scaleway most definitely doesn't follow issues here

**Comment by guillaumedsde** [comment]
Aye, probably not, I've tried the support, they don't seem to care much either

**Comment by kunthar** [comment]
ignorance starts with ignoring the main users of your service ignorant blinds.

**Comment by tbillon** [comment]
This is definitively a mistake for `CONFIG_SQUASHFS_XZ` has been enabled since the [4.4.70](https://github.com/scaleway/kernel-tools/blob/a6330706aa91d30f8d58589e6bf977d231bf1eb3/x86_64/4.4.70-std/.config#L3644) kernel. Really sorry about that, we'll fix it asap.

**Comment by kunthar** [comment]
it would be great to enlighten us when the fix becomes usable. thank you.

**Comment by tbillon** [comment]
`CONFIG_SQUASHFS_XZ` has been re-enabled on the latest kernel builds (4.4.86 & 4.9.47). Sorry for the inconvenience.

**Comment by kunthar** [comment]
could you please kindly redirect us to relevant documentation, how we can select these kernels on installation? current kernel for xenial is `Linux scw-5f26a5 4.4.84-mainline-rev1 #1 SMP Fri Aug 25 00:43:11 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`

**Comment by tbillon** [comment]
The documentation is available here https://www.scaleway.com/docs/bootscript-and-how-to-use-it/

**Comment by kunthar** [comment]
snap installed without error but we had a several issues with conjure-up and we gave up to test. conjure-up installation terminates with `error: cannot perform the following tasks: - Run configure hook of "conjure-up" snap if present (run hook "configure": cannot change profile for the next exec call: No such file or directory)` error. several solutions  tested but no avail. both 4.4.87 rev1 and 4.9.48 rev1 tested. we had to return gce for now.
